cmd/aws-iam-authenticator/add.go

```diff
@@ -0,0 +1,196 @@
+/*
+Copyright 2021 by the contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/manifoldco/promptui"
+	"github.com/spf13/cobra"
+	"k8s.io/client-go/kubernetes"
+	restclient "k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	clientcmd_api "k8s.io/client-go/tools/clientcmd/api"
+	"sigs.k8s.io/aws-iam-authenticator/pkg/config"
+	"sigs.k8s.io/aws-iam-authenticator/pkg/mapper/configmap/client"
+	"sigs.k8s.io/yaml"
+)
+
+var addCmd = &cobra.Command{
+	Use:   "add",
+	Short: "add IAM entity to an existing aws auth configmap",
```

Comment:
Can we be more explicit in the description that makes it clear this is currently only supported for the configmap backend.  i.e. "Note: this is currently ONLY supported..."